app/subscriber.py

```diff
@@ -58,7 +56,28 @@ def receipt_to_case(message: Message):
     log = log.bind(case_id=case_id, created=time_obj_created, tx_id=tx_id)
 
     metadata['response_datetime'] = time_obj_created
-    send_message_to_rabbitmq(json.dumps(metadata))  # NB: in the future this should hold `questionnaire_id`
+
+    receiptMessage = {}
+    eventObject = {}
+    payloadObject = {}
+    responseObject = {}
+
+    eventObject['type'] = 'RESPONSE_RECEIVED'
+    eventObject['source'] = 'RECEIPT_SERVICE'
+    eventObject['channel'] = 'EQ'
+    eventObject['dateTime'] = time_obj_created
+    eventObject['transactionId'] = tx_id
+
+    responseObject['caseId'] = case_id
+    responseObject['questionnaireId'] = questionnaire_id
+    responseObject['unreceipt'] = 'false'
+
+    payloadObject['response'] = responseObject
+
+    receiptMessage['event'] = eventObject
+    receiptMessage['payload'] = payloadObject
+
+    send_message_to_rabbitmq(json.dumps(receiptMessage))  # NB: in the future this should hold `questionnaire_id`
```

Comment:
remove the comment on this line as its not applicable anymore

Reply:
done